examples/quantum_annealing/price_optimization/QUBO-Pricing.ipynb

```diff
@@ -0,0 +1,2855 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "aed8d388",
+   "metadata": {},
+   "source": [
+    "We start by importing Amazon braket SDK and Dwave SDK to be used for quantum annealing in this use case. In addition, we import sklearn to fit the demand estimation model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "1f31ea86",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Requirement already satisfied: sklearn in /home/ec2-user/anaconda3/envs/Braket/lib/python3.7/site-packages (0.0)\n",
+      "Requirement already satisfied: scikit-learn in /home/ec2-user/anaconda3/envs/Braket/lib/python3.7/site-packages (from sklearn) (1.0.1)\n",
+      "Requirement already satisfied: scipy>=1.1.0 in /home/ec2-user/anaconda3/envs/Braket/lib/python3.7/site-packages (from scikit-learn->sklearn) (1.5.2)\n",
+      "Requirement already satisfied: numpy>=1.14.6 in /home/ec2-user/anaconda3/envs/Braket/lib/python3.7/site-packages (from scikit-learn->sklearn) (1.19.2)\n",
+      "Requirement already satisfied: threadpoolctl>=2.0.0 in /home/ec2-user/anaconda3/envs/Braket/lib/python3.7/site-packages (from scikit-learn->sklearn) (3.0.0)\n",
+      "Requirement already satisfied: joblib>=0.11 in /home/ec2-user/anaconda3/envs/Braket/lib/python3.7/site-packages (from scikit-learn->sklearn) (1.1.0)\n"
+     ]
+    }
+   ],
+   "source": [
+    "!pip install sklearn"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "6718da7a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from braket.aws import AwsDevice\n",
+    "from braket.ocean_plugin import BraketSampler, BraketDWaveSampler\n",
+    "from pyqubo import Binary\n",
+    "# magic word for producing visualizations in notebook\n",
+    "%matplotlib inline\n",
+    "import time\n",
+    "import reprlib\n",
+    "from collections import defaultdict\n",
+    "from itertools import combinations\n",
+    "import math\n",
+    "import networkx as nx\n",
+    "import dwave_networkx as dnx\n",
+    "import minorminer\n",
+    "import dimod\n",
+    "from dimod.binary_quadratic_model import BinaryQuadraticModel\n",
+    "from dwave.system.composites import EmbeddingComposite\n",
+    "import numpy as np\n",
+    "from sklearn.linear_model import LinearRegression\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "from qubo_dynamic_pricing import optimize\n",
+    "\n",
+    "np.random.seed(0)\n",
+    "repr_compact = reprlib.Repr()\n",
+    "repr_compact.maxother=200"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "61be0cc9",
+   "metadata": {},
+   "source": [
+    "__NOTE__: Enter your S3 bucket and key below. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "ce15ec79",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Enter the S3 bucket you created during onboarding in the code below\n",
```

Comment:
Braket now supports default s3 buckets, so this step is no longer required. See other changes below reflecting this. I pushed some edits to your script to reflect this as well.